packages/circuits/test/utils.ts

```diff
@@ -648,7 +648,6 @@ const genReputationCircuitInput = async (
         epoch: epoch,
```

Comment:
Why we push utils.ts in test?  Do we reuse method in this file.
if we reuse I think it more sense if we move it to `src` or rename it to index.ts.

Reply:
because it is just used for tests and it will not be used in @unirep/contracts and @unirep/core anymore.